cmd/operator/deploy/rule-evaluator/02-rolebinding.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Google LLC
```

Comment:
I added the kubelet/cAdvisor target to GMP. It looks to be related. Thank you for pointing out, I couldn't why this RBAC error happend.
```
- job_name: kubelet/cadvisor
          ...
          kubernetes_sd_configs:
            - role: pod
              kubeconfig_file: ""
              follow_redirects: true
              enable_http2: true
              selectors:
                - role: pod
                  field: spec.nodeName=$(NODE_NAME)
```

Reply:
Ok.

Though I don't know what this would accomplish. The rule-evaluator isn't scraping any targets, it's just evaluating Prometheus queries against the GMP backend and recording/alerting off of those.

My understanding for the expanded RBAC was to allow the standalone rule-evaluator to discover any Prometheus Alertmanagers in the cluster through Prometheus service discovery, which the roles you're adding should allow for.

It _is still_ a valid config as far the rule-evaluator is concerned, as it's able to parse any Prometheus configuration.

Can you elaborate why you're using this in the rule-evaluator?